src/services/utilities.ts

```diff
@@ -1358,4 +1358,115 @@ namespace ts {
             diagnostics: error ? concatenate(diagnostics, [error]) : diagnostics
         };
     }
-}
+
+    export function getMissingAbstractMembersInsertion(classDecl: ClassDeclaration, resolvedType: ResolvedType, checker: TypeChecker, newlineChar: string): string {
+        const classSymbol = checker.getSymbolOfNode(classDecl);

```

Comment:
Would `getSymbolOfNode` return `undefined`? you may need to check before you call `classSymbol.members` next line